libraries/mbed/targets/cmsis/TARGET_NXP/TARGET_LPC81X/system_LPC8xx.h

```diff
@@ -30,6 +30,8 @@ extern "C" {
 
 #include <stdint.h>
 
+//WH
```

Comment:
You're right. Copy&paste leftover. Should be deleted.

 

  _____  

From: Martin Kojtal [mailto:notifications@github.com] 
Sent: woensdag 28 januari 2015 12:19
To: mbedmicro/mbed
Cc: Wim
Subject: Re: [mbed] Fix and enable the PLL for the LPC812 (#870)

 

In libraries/mbed/targets/cmsis/TARGET_NXP/TARGET_LPC81X/system_LPC8xx.h
<https://github.com/mbedmicro/mbed/pull/870#discussion_r23680268> :

> @@ -30,6 +30,8 @@ extern "C" {
>  
>  #include <stdint.h>
>  
> +//WH

Why are you leaving this comments in the code? We can "blame" to get that
detail.

-
Reply to this email directly or view it on
<https://github.com/mbedmicro/mbed/pull/870/files#r23680268>  GitHub.
<https://github.com/notifications/beacon/AKMUNKqX_u0vErDI96J_pWPPYOUQ_197ks5
nmL0ogaJpZM4DX95z.gif>